src/bin/pgcopydb/ld_transform.c

```diff
@@ -1579,6 +1662,12 @@ FreeLogicalTransaction(LogicalTransaction *tx)
 	tx->first = NULL;
 }
 
+void
+FreeLogicalMessageRelation(LogicalMessageRelation *table)
+{
+	free(table->nspname);
+	free(table->relname);
+}
```

Comment:
Missing function level comment. I believe the function `PQfreemem` should be used on these pointers?

Reply:
Apparently it depends where they come from... so maybe we need to track the information with a new slot in the LogicalMessageRelation structure: `bool pqMemory` or something?